src/pipewire.c

```diff
@@ -224,7 +229,7 @@ pipewire_remote_destroy (PipeWireRemote *remote)
   /* This check is a workaround for older PW versions */
   if (remote->registry)
     spa_hook_remove (&remote->registry_listener);
-  g_clear_pointer (&remote->registry, pw_proxy_destroy);
+  g_clear_pointer (&remote->registry, pipewire_registry_destroy);
```

Comment:
```suggestion
  g_clear_pointer ((struct pw_proxy **)&remote->registry, pw_proxy_destroy);
```

Reply:
ouch, I would have said I tried that and it didn't build, before adding the helper function. Thanks for suggesting that change. I made that change and I guess you can squash all commits before merging, right? or do you want me to merge them into one and force push?

Reply:
For the future: we also try to prefix the commit message with the area the code touches. In this case `pipewire: ...`.